schema/customer/customer_v1.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://example.com/customer.schema.json",
+  "title": "Customer",
+  "description": "Customer info",
+  "type": "object",
+  "properties": {
+    "customerId": {
+      "description": "The unique identifier for a customer",
+      "type": "string"
+    },
+    "firstName": {"type": "string"},
+    "lastName": {"type": "string"}
+  },
+  "required": [ "firstName" ]
+}
```

Comment:
Maybe we can move the customer schema related files to a test branch or something like that.